examples/bpi-monolith-test-suite/bpi/components/vsm/vsm.ts

```diff
@@ -0,0 +1,43 @@
+import { Agreement } from "../storage/agreement";
+import { Transaction } from "../transactions/transaction";
+import { IStorageComponent } from "../storage/storage.interface";
+import { IWorkgroupComponent } from "../workgroup/workgroup.interface";
+import { IVirtualStateMachineComponent } from "./vsm.interface";
+
```

Comment:
presume workflow will be added as well @ognjenkurtic

Reply:
Yep, upcoming